### PR TITLE
[HIRFUS-27] Develop an E-commerce Store Database

### PR DESCRIPTION
**Pull request by Mirriam6**
I have created my first task on the Databases

### CODE REVIEW

File: submissions/mirriamSikalume/database/foodbag.sql [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	food_type VARCHAR(255) NOT NULL,
+	primary key(id)
+);
+
+CREATE TABLE vendors_table (
+	id INT NOT NULL auto_increment,
+	admin_id INT NOT NULL,
+	food_id INT NOT NULL,
+	name VARCHAR (255) NOT NULL,
+	amount INT NOT NULL,
```
Comment (ToluEnny24) [inline submissions/mirriamSikalume/database/foodbag.sql:15]: INT should be changed to decimal or float

File: submissions/mirriamSikalume/database/foodbag.sql [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	primary key(id), foreign key(food_id) references food_table(id) 
+);
+
+CREATE TABLE admins_table (
+	id INT NOT NULL auto_increment,
+	name VARCHAR (255) NOT NULL,
+	type INT NOT NULL,
```
Comment (ToluEnny24) [inline submissions/mirriamSikalume/database/foodbag.sql:22]: ENUM is better.

File: submissions/mirriamSikalume/database/foodbag.sql [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+	email_address VARCHAR (255) NOT NULL,
+	phone_number INT NOT NULL,
```
Comment (ToluEnny24) [inline submissions/mirriamSikalume/database/foodbag.sql:24]: +234, +91 and all will have issues with INT, VARCHAR(50) is better

File: submissions/mirriamSikalume/database/foodbag.sql [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+	password VARCHAR (255) NOT NULL,
+	primary key(id)
+);
+
+CREATE TABLE customers_table (
+	id INT NOT NULL auto_increment,
+	name VARCHAR (255) NOT NULL,
+	phone_number INT NOT NULL,
```
Comment (ToluEnny24) [inline submissions/mirriamSikalume/database/foodbag.sql:32]: Take note as above.

File: submissions/mirriamSikalume/database/foodbag.sql [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+	address VARCHAR (255) NOT NULL,
+	payment_method VARCHAR (255) NOT NULL,
```
Comment (ToluEnny24) [inline submissions/mirriamSikalume/database/foodbag.sql:34]: Its optional, NOT NULL is wrong

**Review by ToluEnny24 — CHANGES_REQUESTED** [review]
Effect the requested changes @Mirriam6

**Comment by Mirriam6** [comment]
I have modified my task, kindly review.

**Review by ToluEnny24 — APPROVED** [review]

**Pull request merged**